diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -7,7 +7,7 @@
     "FILE_NAME_MATCH_CLASS",
     "KDOC_WITHOUT_RETURN_TAG",
     "KDOC_WITHOUT_PARAM_TAG",
-    "MatchingDeclarationName",
+    "MatchingDeclarationName"
```

Comment:
better to leave a comma at the end. it will avoid change in this line in case of adding a new element at the end

Reply:
Done.